Tasks/UseDotNetV2/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 169,
-        "Patch": 3
+        "Minor": 171,
```

Comment:
@vineetmimrot Tejas also updated the version yesterday I think. Please check that.